support-frontend/assets/pages/contributions-landing/checkoutFormIsSubmittableActions.js

```diff
@@ -119,6 +120,7 @@ function enableOrDisableForm() {
 
     const v2RecaptchaCheck =
       window.guardian.recaptchaV2
+      && [Stripe, ExistingCard].includes(state.page.form.paymentMethod)
```

Comment:
@tomrf1 @JoemG @jlieb10 Is it better to do:

`&& state.page.form.stripeCardFormData.formComplete` ?

Reply:
I would still check paymentMethod is `Stripe`. Not sure the `ExistingCard` check is necessary here, we don't do it elsewhere.